Script update

FurTactics opened the pull request:
Hey. I've been looking for something like this for a long time and finally found one of the forks of your project by Cihan Tuncer (https://greasyfork.org/en/scripts/398310). Unfortunately, I found your source code too late, after I had already finished working on his code. 

I noticed that you also have the option to remove banners or something like that. I would really like to share my script with you, and maybe we could combine the two scripts. What I did was improve the stability of speed selection when refreshing the page, going back in history, or navigating within the page after it loads (as we know, YouTube works on the SPA principle, which complicates things a bit). I was only able to implement this by adding several observers to monitor link changes and video appearances. I don't know if it's possible to improve the performance of observers, but I tried to simply reduce the DOM that will be viewed by it. 

P.S. I wanted to try running your script, but it didn't work for me. I only got as far as the logs:
```
[TechoTom custom speeds] Check #1 for target element
[TechoTom custom speeds] success with strategy: noId
```

Your project was found by me, which I'm really glad about. I'm keen to work together!

FurTactics committed:
Increased stability and perfomance

- Merge the changes with the fork created by Cihan Tuncer (https://greasyfork.org/en/scripts/398310)
- Update the observers to monitor URL changes and the appearance of videos
- Add a new style to a remember button

tomsaleeba commented:
Hey @FurTactics, thanks for sharing your work 💪. I didn't realise some of my code ended up on GreasyFork, but that's great that it's helping people.

I don't have time to process your proposed changes right now, but at some stage I'll have a look through and see what I can integrate. Thanks 👍

FurTactics committed:
Update version and enhance playback speed button styles

Updated version number and improved button styling for playback speed control bcs of updated style of youtube

FurTactics committed:
Bump version and enhance live button handling

The version number has been updated and functionality has been added to handle live playback. While loading live video, if no playback buttons are clicked, the speed remains at 1x, but the previous speed for normal videos is still saved. The playback speed buttons can be used as normal in non-live mode and after clicking the live button, the playback speed will revert to 1x
